test/dotnet/dotnet_helpers.sh

```diff
@@ -12,6 +12,9 @@ fetch_latest_version_in_channel() {
         wget -qO- "https://dotnetcli.azureedge.net/dotnet/Runtime/$channel/latest.version"
     elif [ "$runtime" = "aspnetcore" ]; then
         wget -qO- "https://dotnetcli.azureedge.net/dotnet/aspnetcore/Runtime/$channel/latest.version"
+    elif [ "$channel" = "LTS" ]; then
+        echo $(curl -s https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json |  
```

Comment:
I am thinking it would be good to replace all usages of `latest.version` with `release-index.json` for consistency, and because using the `latest.version` file in scripts is not endorsed by the .NET team